phosphor-power-supply/power_supply.cpp

```diff
@@ -499,6 +499,25 @@ void PowerSupply::analyzeMFRFault()
     {
         if (mfrFault < DEGLITCH_LIMIT)
         {
+            if (statusMFR & 0x80)
+            {
+                // Report current share warning once per boot
+                if (!currentShareWarningReported)
+                {
+                    log<level::INFO>(
+                        fmt::format("{} 12V current share warning: "
+                                    "STATUS_WORD = {:#06x} "
+                                    "STATUS_MFR_SPECIFIC = {:#04x}",
+                                    shortName, statusWord, statusMFR)
+                            .c_str());
+                    currentShareWarningReported = true;
```

Comment:
i wonder if you want to put this into clearFaultFlags() so it would trace it again on the next power cycle, like it does with all the other ones.

Reply:
Added currentShareWarningReported to clearFaultFlags